get-account-info-using-portal-api-keys/README.md

```diff
@@ -0,0 +1,19 @@
+# Get Account Info Using HEDERA Portal API Keys
+
+This script shows how to retrieve account information using the Hedera Portal API Keys (Personal Access Tokens). The script obtains the account information from the portal programmatically and checks balances using the Hedera SDK.
+
+Developers can use this functionality to automate and manage multiple Hedera accounts programmatically, which can be useful for continous integration (CI) pipelines. This can also be used to more easily obtain account info after testnet resets. Keep in mind that public-private key pairs from the Hedera Portal do not change after reset - only account IDs change.
```

Comment:
```suggestion
Developers can use this functionality to automate and manage multiple Hedera accounts programmatically, which can be useful for continous integration (CI) pipelines. This can also be used to more easily obtain account info after Testnet resets. Keep in mind that cryptographic keys from the Hedera Portal do not change after reset - only account IDs change.
```